### PR TITLE
Clarify government responses on homepage

### PR DESCRIPTION
**Pull request by henryhadlow**
* Add 'The government responded...' to response quotes
* Remove headings 'Most recent government responses / debates in Parliament'
* Change 'Read the government response' to 'Read the response in full'

### CODE REVIEW

**Comment by yahoopete** [comment]
LGTM. 

Wonder if we said HM Government responded might give it oomph...?

**Comment by henryhadlow** [comment]
I haven't added your timestamp yet – wasn't that convinced.
@pixeltrix seemed to be having a couple issues with this PR :$

**Comment by alexander-bobin** [comment]
Is this ready to be merged?

File: app/views/pages/_home_debated_petitions.html.erb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <% if actioned[:with_debate_outcome][:count].zero? %>
   <p>This Parliament has not debated any petitions yet.</p>
 <% else %>
-  <h2>Most recently debated in Parliament</h2>
   <ol class="threshold-petitions debate-petitions-list">
```
Comment (alexander-bobin) [inline app/views/pages/_home_debated_petitions.html.erb]: `.debate-petitions-list` is no longer used in our CSS. Can we remove it?

File: app/views/pages/_home_responded_petitions.html.erb [hunk 2]
```diff
@@ -1,13 +1,13 @@
 <% if actioned[:with_response][:count].zero? %>
   <p>This Government has not responded to any petitions yet.</p>
 <% else %>
-  <h2>Most recent government responses</h2>
   <ol class="threshold-petitions response-petitions-list">
```
Comment (alexander-bobin) [inline app/views/pages/_home_responded_petitions.html.erb]: `.response-petitions-list` is no longer used in our CSS. Can we remove it?

File: app/assets/stylesheets/petitions/views/_home.scss [hunk 3]
```diff
@@ -56,11 +56,19 @@
 .threshold-panel:hover, .threshold-panel:focus {
   background-color: darken($petitions-panel-colour, 5%);
 }
-
 .response-threshold, .debate-threshold {
   background-position: 15px 15px;
   background-repeat: no-repeat;
 }
+.pull-quote {
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/views/_home.scss]: Can we wrap `.pull-quote` and `.response-intro` inside a `.threshold-petitions` so it is more specific? There is no obvious relationship here and these will apply across the site.

**Pull request merged**